superset/reports/commands/execute.py

```diff
@@ -403,6 +407,7 @@ def __init__(
         self._session = session
         self._report_schedule = report_schedule
         self._scheduled_dttm = scheduled_dttm
+        self._execution_id = uuid4()
```

Comment:
Have you checked if we can use celery's UUID, and pass it into the `AsyncExecuteReportScheduleCommand` constructor?

Reply:
The closest thing I found to this is [this Github issue](https://github.com/celery/celery/issues/1813), but it only described how it could be done when triggering tasks with `delay` or `apply_async`, and I wasn't sure how to do it for a task triggered by a beat schedule.

Reply:
Actually scratch that, I tweaked my Google search and found [this](https://stackoverflow.com/questions/18872854/getting-task-id-inside-a-celery-task), let me see if I can get it to work.

Reply:
The `delay` is invoked by the caller.

use `<job_name>.request.id` to get the id, then use it here:
https://github.com/apache/superset/blob/master/superset/tasks/scheduler.py#L67

on this case should be `execute.request.id`

Reply:
check if you need to use the `bind=True` and then `self.request.id` to bind the function to the task class.
https://docs.celeryproject.org/en/latest/userguide/tasks.html?highlight=requestcontext#basics

Reply:
Yep, `execute.request.id` worked!